lnwallet/chainfee/estimator.go

```diff
@@ -593,8 +600,12 @@ func (w *WebAPIEstimator) EstimateFeePerKW(numBlocks uint32) (SatPerKWeight, err
 	}
 
 	feePerKb, err := w.getCachedFee(numBlocks)
+
+	// If the estimator doesn't have enough data, or returns an error, then
+	// to return a proper value, then we'll return the default fall back
```

Comment:
nit: I believe the "then to return a proper value" part is just leftover from a previous comment version.

Reply:
Fixed!